src/platformdirs/android.py

```diff
@@ -56,6 +56,24 @@ def user_log_dir(self) -> str:
             path = os.path.join(path, "log")
         return path
 
+    @property
+    def user_documents_dir(self) -> str:
+        """
+        :return: documents directory tied to the user e.g. ``/storage/emulated/0/Documents``
+        """
+        # Get directories with pyjnius
```

Comment:
This should be cached the same way we do for the other jnius access.

Reply:
I don't have any caching experience. Is [this](https://github.com/platformdirs/platformdirs/pull/39/commits/bc3449b60105c9306526558704e759f7eddf44dd) alright?